landoapi/models/landing.py

```diff
@@ -63,14 +63,18 @@ def create(cls, revision_id, phabricator_api_key=None, diff_id=None):
         # save landing to make sure we've got the callback
         landing = cls(revision_id=revision_id, diff_id=diff_id).save()
 
-        trans = TransplantClient()
-        callback = '%s/landings/%s/update' % (
-            os.getenv('HOST_URL'), landing.id
+        # Define the pingback URL with the port
+        callback = '{host_url}:{pingback_port}/landings/{id}/update'.format(
```

Comment:
this issue should have blocked this from merging - the callback url as implemented won't work.  please ensure this is tracked and resolved before release.